### ISSUE TITLE
False positive for broken autoloading

### ISSUE THREAD

**Issue by FlyingDR** [kalessil/phpinspectionsea#1601]
| Subject        | Details                                                                       |
| :------------- | :---------------------------------------------------------------------------- |
| Plugin         | Php Inspections (EA Ultimate), v3.0.10 |
| Language level | Tested with PHP 7.2 and 7.4 |

Steps to reproduce
--
1. Create new Symfony project, using [Symfony CLI](https://symfony.com/download) and `symfony new test-project`.
2. Open project into PHPStorm, observe that `composer.json` contains default autoloading configuration:
```
"autoload": {
    "psr-4": {
        "App\\": "src/"
    }
},
"autoload-dev": {
    "psr-4": {
        "App\\Tests\\": "tests/"
    }
},
```
3. Create create new PHP class that will:
   1. Reside in `src` (and hence will have `App\` as first namespace component) 
   2. Will have some intermediate namespace components
   2. Will have `App` as last namespace component

Example class FQCN: `App\Event\App\MyEvent`, but name of the class itself, names and amount of intermediate components doesn't matter.

4. Open newly created class in editor, observe no errors reported for this class
5. Open `composer.json` and remove `autoload-dev` section
6. Open class file in editor

Current behaviour
--
There is error reported for the class:
```
[EA] Class autoloading might be broken: directory path and namespace are not matching. Looked up: src/Event/App/MyEvent.php, for */EventMyEvent.php 
```

Expected behaviour
--
No errors should be reported

Environment details
---
PHPStorm 2020.1.3 on Windows 7

**Comment by kalessil** [comment]
Thank you for reporting @FlyingDR, I was able to reproduce the problem and looking how to fix it.

**Comment by kalessil** [comment]
Fixed!

**Comment by FlyingDR** [comment]
Thank you!
